src/Libraries/RevitNodes/Elements/DirectShape.cs

```diff
@@ -0,0 +1,403 @@
+﻿using System;
+
+using Autodesk.Revit.DB;
+
+using DynamoServices;
+
+using Revit.GeometryConversion;
+
+using RevitServices.Persistence;
+using RevitServices.Transactions;
+
+using Point = Autodesk.DesignScript.Geometry.Point;
+using Autodesk.DesignScript.Geometry;
+using System.Collections.Generic;
+using RevitServices.Materials;
+
+namespace Revit.Elements
+{
+    /// <summary>
+    /// A Revit DirectShape, which is a wrapper for some other geometrical entities 
+    /// </summary>
+    [RegisterForTrace]
+    public class DirectShape : Element
+    {
+        
+        #region Private Properties
+
+        private static Guid DYNAMO_DIRECTSHAPE_APP_GUID
+        {
+            get
+            {
+                return Guid.Parse("03deaabe-6989-4d20-ba68-557001d2f45c");
```

Comment:
Use a static constructor for building reference types whose value never changes.